### PR TITLE
 [ Cloud Pak 3.0] - Move certificates from v1alpha1 to v1

### PR DESCRIPTION
**Pull request by rashmi43**
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/55613

### CODE REVIEW

File: pkg/controller/authentication/certificate.go [hunk 1]
```diff
@@ -18,8 +18,10 @@ package authentication
 
 import (
 	"context"
-
+	certmgrv1 "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
```
Comment (Tirumalavasa) [inline pkg/controller/authentication/certificate.go:21]: This requires go.mod changes to pull in cert-manager apis

**Comment by rwhundley** [comment]
Based upon the build failures, we should go ahead and perform the same replacement of `Handler` for `ProbeHandler` where appropriate.

**Comment by rwhundley** [comment]
Also, I don't know if you're performing development on different boxes and that's why some of these commits don't have signatures, but I can help you get that fixed so that we have all commits getting signed properly.

**Review by Tirumalavasa — APPROVED** [review]
/lgtm

**Pull request merged**